Gemfile

```diff
@@ -50,7 +50,7 @@ gem 'laa-criminal-applications-datastore-api-client',
 
 gem 'laa-criminal-legal-aid-schemas',
     github: 'ministryofjustice/laa-criminal-legal-aid-schemas',
-    tag: 'v1.1.15'
+    ref: '6a59d8ac5f00579bb7f536da2e1dfba4b0577e1b'
```

Comment:
Is this right?

Reply:
@RoseSAK Updated schema version to v1.1.16